src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
```

Comment:
The method `convertToTransaction` should not begin with an empty line according to the checklist.